README.md

```diff
@@ -71,5 +72,5 @@ This repository contains all submitted Centrifuge Proposals. The genesis proposa
 ## Proposed CPs
 |CP | Short title |
 |---|-------------|
-| [CP117](./cps/CP117.md)  | Give a mandate to form the Treasury Advisory Group   |
+
```

Comment:
There is an extra line break here, preventing the table from being rendered properly.